Doc/reference/datamodel.rst

```diff
@@ -1714,9 +1714,9 @@ define :meth:`__get__`, then accessing the attribute will return the descriptor
 object itself unless there is a value in the object's instance dictionary.  If
 the descriptor defines :meth:`__set__` and/or :meth:`__delete__`, it is a data
 descriptor; if it defines neither, it is a non-data descriptor.  Normally, data
-descriptors define both :meth:`__get__` and :meth:`__set__`, while non-data
+descriptors define both :meth:`__get__` and :meth:`__set__` (and/or :meth:`__delete__`), while non-data
```

Comment:
This line should be left unchanged.  It talks about the *normal* usage and it is correct.  Also, there is no need to belabor the point about ``__delete__()``.

Reply:
Okay, I thought the normal usage was insisting on the `__get__` addition to a data descriptor (whatever its kind, `__set__`, `__delete__` or both), not on the addition plus the kind of data descriptor. Updated.